x/lockup/client/cli/query.go

```diff
@@ -42,7 +42,7 @@ func GetQueryCmd() *cobra.Command {
 		GetCmdAccountLockedLongerDurationNotUnlockingOnly(),
 		GetCmdAccountLockedLongerDurationDenom(),
 		GetCmdOutputLocksJson(),
-		GetCmdSyntheticLockupsByLockupID(),
+		GetCmdSyntheticLockupByLockupID(),
```

Comment:
Suggest avoiding breaking CLI and instead adding a new CLI

Reply:
Added back the old CLI cmd and added deprecated in the description https://github.com/osmosis-labs/osmosis/pull/5265/commits/7bd39552716c16466babca5293a76c39b5fd714e